.github/workflows/labeler-review.yml

```
@@ -7,23 +7,22 @@ permissions:
   pull-requests: write
   issues: write
   contents: read
+  organization: read
```

actionlint (1.7.9)

unknown permission scope "organization". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)